styles.css

```diff
@@ -0,0 +1,30 @@
+@import url('https://fonts.googleapis.com/css2?family=Domine&family=Montserrat:wght@100&display=swap');
+
+body {
+  background-color: #6f23ff;
+  font-family: 'Montserrat', sans-serif;
+  margin: 50px 100px 50px 100px;
+  color: white;
+  text-align: center;
+}
+h1{
+  font-family: 'Domine', serif;
+}
+
+h2{
+  padding-top: 40px;
+  border-top: 0.5px solid white;
+}
+
+ol, ul{
+  display: inline-block;
+  text-align: left;
+}
+
+#bio a{
+  color: aqua;
+}
+
+.social-media-links{
+  color: #ffd540;
+}
```

Comment:
I kindly recommend you to improve the indentation of the CSS file.

Reply:
- [ ] Your comment appears to be polite. However, one thing I would suggest you include is an explanation of 'why' he/she should improve the indentation. :ok_hand: